src/components/AboutContainer/AboutContentContainer/AboutContentContainer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import useStyles from './AboutContentContainer.styles';
+import { WhatWeDo, WhoWeAre, Programs } from './AboutContentPages';
+import { PropTypes } from 'prop-types';
+
+const AboutContentContainer = ({ selected }) => {
+  const { container, content } = useStyles();
+
+  const page = () => {
+    let result;
+    switch (selected) {
+      case 'What We Do':
+        result = <WhatWeDo />;
+        break;
+      case 'Who We Are':
+        result = <WhoWeAre />;
+        break;
+      case 'Programs':
+        result = <Programs />;
+        break;
+      default:
+        <WhatWeDo />;
+    }
+    return result;
+  };
```

Comment:
let's rewrite this.. get rid of the variable result and the break statement... return the component

Reply:
Good idea, refactored in b261349176882a861ea91d2a5d4ec317ff829e91